File: Part2/C/BasicConcepts/BasicConcepts_InvalidASCII.c
#include <stdio.h>

int main()
{
    // So the ASCII table shows the tab symbol as TAB, but this doesn't work in programming.
    // I think in some programs this will crash, so it will be in a separate program.
    char charInvalid = 'TAB';
    printf ( "Invalid char: %c\n", charInvalid );

    return 0;
}

File: Part2/Go/BasicConcepts/BasicConcepts_InvalidASCII.go
package main

import "fmt"

func main() {
	// So the ASCII table shows the tab symbol as TAB, but this doesn't work in programming.
	// I think in some programs this will crash, so it will be in a separate program.
	// Go doesn't allow multi-character rune literals - this will cause compile error
	charInvalid := 'TAB'
	fmt.Printf ( "Invalid char: %c\n", charInvalid )
}
